### PR TITLE
Zeta 425

### PR DESCRIPTION
**Pull request by picojp32**

### CODE REVIEW

File: libs/employees/src/lib/employees/employees.component.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <h1>Employees</h1>
 
 <razroo-fully-architected-dashboard-data-table
-  [tableData]="employees$ | async"
+  [tableData]="employees$| async"
```
Comment (CharlieGreenman) [inline libs/employees/src/lib/employees/employees.component.html:4]: Keep the space between the employees$ and |

File: libs/employees/src/lib/employees/employees.component.ts [hunk 2]
```diff
@@ -13,7 +13,9 @@ export class EmployeesComponent implements OnInit {
 
   constructor(
     private employeesService: EmployeesService
-  ) {}
+  ) {
+    this.employeesService.getEmployees().subscribe()
```
Comment (CharlieGreenman) [inline libs/employees/src/lib/employees/employees.component.ts:17]: This should go inside of ngOnInit to confirm to Angular lifecycle

File: libs/tickets/src/lib/tickets/tickets.component.ts [hunk 3]
```diff
@@ -39,17 +39,19 @@ const ELEMENT_DATA = [
...
   elementData;
+
   tickets$ = this.ticketsService.getTickets();
 
 
   constructor(
     private ticketsService: TicketsService
-  ) { }
+  ) {
+      this.ticketsService.getTickets().subscribe()
```
Comment (CharlieGreenman) [inline libs/tickets/src/lib/tickets/tickets.component.ts:49]: Same. Inside of ngOnInit

Comment (CharlieGreenman) [reply libs/tickets/src/lib/tickets/tickets.component.ts:49]: Add a console log for data

**Review by CharlieGreenman — CHANGES_REQUESTED** [review]

File: libs/employees/src/lib/employees/employees.component.html (hunk 1 above)
Comment (picojp32) [reply libs/employees/src/lib/employees/employees.component.html:4]: Fixed

File: libs/employees/src/lib/employees/employees.component.ts (hunk 2 above)
Comment (picojp32) [reply libs/employees/src/lib/employees/employees.component.ts:17]: Fixed

File: libs/tickets/src/lib/tickets/tickets.component.ts (hunk 3 above)
Comment (picojp32) [reply libs/tickets/src/lib/tickets/tickets.component.ts:49]: Fixed

**Review by CharlieGreenman — APPROVED** [review]

**Pull request merged**